test/validations.test.js

```diff
@@ -614,7 +614,55 @@ describe('validations', function() {
   });
 
   describe('exclusion', function() {
-    it('should validate exclusion');
+    it('fails when excluded value is used for property', function(done) {
+      User.validatesExclusionOf('name', {in: ['bob']});
+      User.create({name: 'bob'}, function(err, user) {
+        err.should.be.instanceof(Error);
+        err.details.messages.should.match({name: /is reserved/});
+        done();
+      });
+    });
+
+    it('passes when excluded value not found for property', function(done) {
+      User.validatesExclusionOf('name', {in: ['dude']});
+      User.create({name: 'bob'}, function(err, user) {
+        if (err) return done(err);
+        user.name.should.eql('bob');
+        done();
+      });
+    });
+
+    it('fails with a custom error message', function(done) {
+      User.validatesExclusionOf('name', {in: ['bob'], message: 'cannot use this'});
+      User.create({name: 'bob'}, function(err) {
+        err.should.be.instanceof(Error);
+        err.details.messages.should.match({name: /cannot use this/});
+        done();
+      });
+    });
+
+    it('fails with a null value when allowNull is false', function(done) {
+      User.validatesExclusionOf('name', {in: ['bob'], allowNull: false});
+      User.create({name: null}, function(err) {
+        err.should.be.instanceof(Error);
+        err.details.messages.should.match({name: /is null/});
+        done();
+      });
+    });
+
+    it('passes with a null value when allowNull is true', function(done) {
+      User.validatesExclusionOf('name', {in: ['bob'], allowNull: true});
+      User.create({name: null}, done);
```

Comment:
Well I disagree, `done` also receives the `err` and `res` parameters. So it is equivalent to this.
```
 User.create({name: null}, function(err, res) {
        done(err, res);
      });
```
And if `err` is not null, then mocha will detect it and flag the test as fail. You can test this by setting `allowNull` to false, the test fails with `The User instance is not valid. Details: name is null`. So I maintain that this is covered by the test.